Exercises/JavaScript/Ex 18/Index.html

```diff
@@ -0,0 +1,11 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lesson 18</title>
+    <script src="About.js"></script>
+</head>
+<body>
+    <h1>This is Home Page</h1>
+</body>
+</html>
```

Comment:
You have external js code, that's good.
But where is the file with js code inside html page?